src/dataflow/components/dataflow-program-toolbar.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./dataflow-base";
+import { observer, inject } from "mobx-react";
```

Comment:
You can get rid of `BaseComponent`, `IBaseProps`, `observer`, and `inject` as well.

Reply:
Right.  Fixed.